docs/source/faq/index.rst

```diff
@@ -80,24 +97,225 @@ of these popular operating systems from the past few years.
-:doc:`load it in a few lines of code. </user_guide/dataset_creation/datasets>`.
+Yes! Here's a couple options:
 
-Alternatively, you could launch a :ref:`remote session <remote-data>` of the
-FiftyOne App on your machine that another user can connect to from their local
-machine. This workflow does require that both users have the
+**Option 1: Export and share**
+
+You can easily :ref:`export a dataset <exporting-datasets>` in one line of
+code, zip it, and share the zip with your collaborator, who can then
+:ref:`load it in a few lines of code <loading-datasets-from-disk>`.
+
+**Option 2: Sharing a remote session**
+
+Alternatively, :ref:`see this FAQ <faq-multiple-sessions-same-dataset>` for
+instructions on launching a remote session and inviting collaborator(s) to
+connect to it from their local machines.
+
+.. _faq-brain-closed-source:
 
 Are the Brain methods open source?
 ----------------------------------
 
-No. Although the :ref:`core library <https://github.com/voxel51/fiftyone>` is
-open source and the :doc:`Brain methods </user_guide/brain>` are freely
-available for use for any commerical or non-commerical purposes, the Brain
-methods are closed source.
+No. Although the `core library <https://github.com/voxel51/fiftyone>`_ is open
+source and the :ref:`Brain methods <fiftyone-brain>` are freely available for
+use for any commerical or non-commerical purposes, the Brain methods are closed
+source.
 
-Check out the :doc:`Brain documentation </user_guide/brain>`` for detailed
+Check out the :ref:`Brain documentation <fiftyone-brain>` for detailed
 instructions on using the various Brain methods.
+
+.. _faq-multiple-sessions-same-dataset:
+
+Can I connect multiple App instances to the same dataset?
+---------------------------------------------------------
+
+Yes, multiple App instances can be connected to the same |Dataset| via remote
+sessions.
+
+.. note:
+
+    Keep in mind that all users must have ssh access to the system from which
+    the remote session(s) are launched in order to connect to them.
+
+    In addition, and all users must have
+    :ref:`FiftyOne installed <installing-fiftyone>` on their local machines in
+    order to launch an App instance.
+
+You can achieve multiple connections in two ways:
+
+**Option 1: Same dataset, multiple sessions**
+
+The typical way to connect multiple App instances to the same dataset is to
+create a separate remote session instance on the machine that houses the
+|Dataset| of interest for each local App instance that you want to create.
+:ref:`See this FAQ <faq-serve-multiple-remote-sessions>` for instructions on
+doing this.
+
+**Option 2: Same dataset, same session**
+
+Another option is to connect multiple App instances to a single remote session.
+
+First, :ref:`create a remote session <remote-session>` on the system that
+houses the |Dataset| using either the CLI or Python:
+
+.. tabs::
+
+  .. group-tab:: CLI
+
+    .. code-block:: shell
```

Comment:
FYI prefer `shell` over `bash` here, since shell is generic and bash is a specific shell. (All docs should be using `shell`)